Add legal IPTV Links for testing (#192)
jellyfin/jellyfin-docs: general/server/live-tv/setup-guide.md

* Auto-loop live streams is sometimes necessary for some IPTV channels.  Turn this on only if your streams are not playing correctly

> [!NOTE]

> > Here is a github for legal samples to use to test connectivity

Reviewer: ```suggestion
> Here is a list of legal samples to use to test connectivity.
```
